multihost_test/Bugzillas/test_regression.py

```diff
@@ -170,3 +170,29 @@ def test_bz_1206273(self, multihost):
         assert cmd1.returncode == 0
         assert cmd3.returncode == 0
         assert user in cmd2.stdout_text
+
+    def test_bz_1089666(self, multihost):
+        """
+        :title: 'usermod -m -d ' should warn that it did not create
+            destination(new)home directory if source home directory does not exist
```

Comment:
It took me some time to understand what you were testing. I'd suggest you to rephrase the title. My suggestion:
Given an existing user with no home directory, when the home directory is moved to a new location, then the new directory won't be created and a warning message will be printed.